client/db/bolt/db.go

```diff
@@ -699,18 +698,21 @@ func (db *BoltDB) UpdateOrder(m *dexdb.MetaOrder) error {
 		return newBucketPutter(oBkt).
 			put(baseKey, uint32Bytes(ord.Base())).
 			put(quoteKey, uint32Bytes(ord.Quote())).
+			put(typeKey, []byte{byte(ord.Type())}).
+			put(orderKey, order.EncodeOrder(ord)).
 			put(statusKey, uint16Bytes(uint16(md.Status))).
 			put(dexKey, []byte(md.Host)).
 			put(updateTimeKey, uint64Bytes(timeNow())).
 			put(proofKey, md.Proof.Encode()).
+			put(lotSizeKey, uint64Bytes(md.LotSize)).
 			put(changeKey, md.ChangeCoin).
 			put(linkedKey, linkedB).
-			put(typeKey, []byte{byte(ord.Type())}).
-			put(orderKey, order.EncodeOrder(ord)).
 			put(swapFeesKey, uint64Bytes(md.SwapFeesPaid)).
+			put(redemptionFeesKey, uint64Bytes(md.RedemptionFeesPaid)).
 			put(maxFeeRateKey, uint64Bytes(md.MaxFeeRate)).
 			put(redeemMaxFeeRateKey, uint64Bytes(md.RedeemMaxFeeRate)).
-			put(redemptionFeesKey, uint64Bytes(md.RedemptionFeesPaid)).
+			put(fromVersionKey, uint32Bytes(md.FromVersion)).
+			put(toVersionKey, uint32Bytes(md.ToVersion)).
```

Comment:
**Bug fix** to extract from this PR. ( `UpdateOrder` was not storing `ToVersion` or `FromVersion`)